### PR TITLE
Fix/ac optional comments

### PR DESCRIPTION
**Pull request by melisabok**
This PR fixes HSDIP process function errors, handling the AC groups as optional. 

Also the meta reviews can be posted by program chairs when there are no ACs set.

### CODE REVIEW

File: openreview/conference/invitation.py [hunk 1]
```diff
@@ -415,12 +417,17 @@ def __init__(self, conference, name, note, start_date, due_date, public):
         content = invitations.meta_review.copy()
 
         readers = ['everyone']
+        regex = conference.get_program_chairs_id()
+        invitees = [conference.get_program_chairs_id()]
+        private_readers = [conference.get_program_chairs_id()]
+
+        if conference.use_area_chairs:
+            regex = conference.get_area_chairs_id(note.number)[:-1] + '[0-9]+'
```
Comment (pmandler) [inline openreview/conference/invitation.py]: What if there are 10 or more ACs?  Too unlikely?

Comment (melisabok) [reply openreview/conference/invitation.py]: + means one or more characters, so it is unlimited.

Comment (pmandler) [reply openreview/conference/invitation.py]: I meant the [:-1] part only strips off one character.

Comment (melisabok) [reply openreview/conference/invitation.py]: That is removing the 's' from the name, like Area_Chairs --> Area_Chair1

**Pull request merged**